### ISSUE TITLE
Search summary statisticts

### ISSUE THREAD

**Issue by mcweenysean** [yjunechoe/Snowglobe#12]
Hey June,

The summary search statistics have been missing for a while - I think since we changed to the new UI. I'm working on this, but if you know where this got removed off-hand and could send me the chunk that would be very helpful.

**Comment by yjunechoe** [comment]
Do you mean the summary statistics with [skimr](https://github.com/ropensci/skimr) from way back? 

I found it to be bit clunky for the user and dropped this in favor of a text version provided by `search_summary_txt`

https://github.com/yjunechoe/Snowglobe/blob/6d4099d650ba132b70fcd9999649dc26818d1857/server.R#L701-L709

It wouldn't be hard to implement back in though. We'd probably have to create a tab dedicated just to the skimr output because it takes up a whole page

**Comment by mcweenysean** [comment]
yeah - I think the most popular journals/authors was really interesting for people and both Sumit and the library (working on getting us funding) have expressed interest in that feature. Could definitely take up a whole tab

**Comment by yjunechoe** [comment]
Oh you mean including the plots too? This is the code for those from the last time I worked on it. 

https://github.com/yjunechoe/Snowglobe/blob/88309911e9d6bc5f8eafbe2b179c46f59759ce32/app.R#L262-L345

We'd have to add this code back into server and make room for rendering them in the ui. I think some variable names changed since then but shouldn't be too difficult to add back in aside from figuring that out.

**Comment by mcweenysean** [comment]
Yeah definitely - I think it's just these that'll need to fixed:

old ```output_data_c()``` I think is equivalent to ```comprehensive_output()```
old ```input_data()``` I think is equivalent to ```staged_file_searched()```
```f.data``` and ```b.data()``` I don't think have current equivalents
